src/OrganizationIntegration/OrganizationIntegrationForm/OrganizationIntegrationForm.js

```diff
@@ -0,0 +1,175 @@
+import React, {
+  useMemo,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { useHistory } from 'react-router';
+
+import stripesForm from '@folio/stripes/final-form';
+import { AppIcon } from '@folio/stripes/core';
+import {
+  AccordionSet,
+  AccordionStatus,
+  Button,
+  checkScope,
+  Col,
+  collapseAllSections,
+  ExpandAllButton,
+  expandAllSections,
+  HasCommand,
+  Pane,
+  PaneFooter,
+  Row,
+} from '@folio/stripes/components';
+
+import {
+  handleKeyCommand,
+} from '@folio/stripes-acq-components';
+
+import { IntegrationInfoForm } from './IntegrationInfoForm';
+import { EdiForm } from './EdiForm';
+import { FtpForm } from './FtpForm';
+import { SchedulingForm } from './SchedulingForm';
+
+const OrganizationIntegrationForm = ({
+  acqMethods,
+  accounts,
+  onClose,
+  paneTitle,
+  handleSubmit,
+  pristine,
+  submitting,
+}) => {
+  const history = useHistory();
+  const accordionStatusRef = useRef();
+  const paneFooter = useMemo(
+    () => {
+      const start = (
+        <FormattedMessage id="ui-organizations.button.cancel">
+          {(btnLabel) => (
+            <Button
+              id="clickable-close-contact-person-footer"
+              buttonStyle="default mega"
+              onClick={onClose}
+            >
+              {btnLabel}
+            </Button>
+          )}
+        </FormattedMessage>
+      );
+
+      const end = (
+        <Button
+          id="clickable-save-contact-person-footer"
+          type="submit"
+          buttonStyle="primary mega"
+          disabled={pristine || submitting}
+          onClick={handleSubmit}
+        >
+          <FormattedMessage id="ui-organizations.button.saveAndClose" />
```

Comment:
why `start` and `end` have different styles for label translation?)

Reply:
because it's copy-paste :) I'll correct it with BE integration + quality gates fixes